quinn-proto/src/config/mod.rs

```diff
@@ -460,6 +460,9 @@ pub struct ClientConfig {
     /// Cryptographic configuration to use
     pub(crate) crypto: Arc<dyn crypto::ClientConfig>,
 
+    /// Validation token store to use
+    pub(crate) token_store: Option<Arc<dyn TokenStore>>,
```

Comment:
Would it be simpler to omit the `Option` and provide an impl that drops everything?